sources/views/contact-info.js

```diff
@@ -0,0 +1,83 @@
+import { JetView } from "webix-jet";
+import { contacts } from "../models/contacts.js";
+import { statuses } from "../models/statuses.js";
+
+export default class ContactInfo extends JetView {
+	config() {
+		const _ = this.app.getService("locale")._;
+		const formButtons = {
+			cols: [
+				{ view: "button", label: _("Delete"), css: "button--style" },
+				{ view: "button", label: _("Edit"), css: "button--style" }
+			]
+		};
+		const userInfo = {
+			view: "template",
+			localId: "userInfo",
+			fillspace: true,
+			template: contact => {
+				return `
+			<div class="user-info-grid">
+			<div class ="contact-info--name">${contact.FirstName} ${contact.LastName}</div>
+			<div class = "user-info-body-grid">
+				<div class="user-info-photo"><img src=${contact.Photo} alt="User photo" /></div>
+				<div class="user-info--col">
+					<div id="user-mail">
+						<span class="webix_icon mdi mdi-email"></span>
+						<span>${contact.Email || "no data"}</span>
+					</div>
+					<div id="user-skype">
+						<span class="webix_icon mdi mdi-skype"></span>
+						<span>${contact.Skype || "no data"}</span>
+					</div>
+					<div id="user-job">
+						<span class="webix_icon mdi mdi-account-card-details"></span>
+						<span>${contact.Job || "no data"}</span>
+					</div>
+					<div id="user-company">
+						<span class="webix_icon mdi mdi-briefcase"></span>
+						<span>${contact.Company || "no data"}</span>
+					</div>
+				</div>
+				<div class="user-info--col">
+					<div id="user-birth">
+						<span class="webix_icon mdi mdi-calendar"></span>
+						<span>${contact.Birthday || "no data"}</span>
+					</div>
+					<div id="user-location">
+						<span class="webix_icon mdi mdi-city"></span>
+						<span>${contact.Address || "no data"}</span>
+					</div>
+				</div>
+			</div>
+			<div><span class="user-info--status">${contact.Status || "no data"}</span></div>
+		</div>
+		</div>`;
+			}
+		};
+
+		const view = { rows: [formButtons, userInfo] };
+
+		return view;
+	}
+
+	init() {
+		this.info = this.$$("userInfo");
+	}
+	urlChange(view, url) {
+		const elementId = url[0].params.id;
```

Comment:
Do not rely on segments of URL unless there are no other options (which is a rare condition).
Use [getParam](https://webix.gitbook.io/webix-jet/api/jetview-api/jetview-methods#this-getparam) instead of direct reference to the URL where it's possible.

Reply:
Fixed.